### PR TITLE
fix: dependecies and helper - get started with embeddings tuning

### PR DESCRIPTION
**Pull request by inardini**
# Description

- Fixing the DocAIParser from LangChain
- Removing unnecessary helper function

### CODE REVIEW

File: embeddings/get_started_with_embedding_tuning.ipynb [hunk 1]
```diff
@@ -1,1438 +1,1438 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "d975e698c9a4"
+      },
+      "source": [
+        "### Objective\n",
+        "\n",
+        "In this tutorial, you learn how to tune the text embedding model, `textembedding-gecko`.\n",
```
Comment (polong-lin) [inline embeddings/get_started_with_embedding_tuning.ipynb:85]: This needs a better explanation/justification of why the user would want to consider tuning their text embedding model. E.g. what problem would this solve for the user?

File: embeddings/get_started_with_embedding_tuning.ipynb [hunk 2]
```diff
@@ -1,1438 +1,1438 @@
...
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8VVac8ah2DzJ"
+      },
+      "source": [
+        "### Retrieve similar items using the tuned embedding model\n",
```
Comment (polong-lin) [inline embeddings/get_started_with_embedding_tuning.ipynb:1297]: This section should compare the results before and after tuning for the user, and show noticeably better results after tuning.

File: embeddings/get_started_with_embedding_tuning.ipynb [hunk 3]
```diff
@@ -1,1438 +1,1438 @@
...
+        "    return client.create_processor(\n",
+        "        parent=parent,\n",
+        "        processor=documentai.Processor(\n",
+        "            display_name=processor_display_name, type_=\"OCR_PROCESSOR\"\n",
+        "        ),\n",
+        "    )\n",
+        "\n",
+        "\n",
+        "def generate_queries(\n",
+        "    chuck: str,\n",
```
[the remark this answers is not in the document]
Comment (inardini) [reply embeddings/get_started_with_embedding_tuning.ipynb:518]: done

File: embeddings/get_started_with_embedding_tuning.ipynb [hunk 4]
```diff
@@ -1,1438 +1,1438 @@
...
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Q2KRaRzHAF8f"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint = aiplatform.Endpoint.create(\n",
```
[the remark this answers is not in the document]
Comment (inardini) [reply embeddings/get_started_with_embedding_tuning.ipynb:1229]: done

File: embeddings/get_started_with_embedding_tuning.ipynb (hunk 1 above)
Comment (inardini) [reply embeddings/get_started_with_embedding_tuning.ipynb:85]: done

File: embeddings/get_started_with_embedding_tuning.ipynb (hunk 2 above)
Comment (inardini) [reply embeddings/get_started_with_embedding_tuning.ipynb:1297]: In the Evaluate the tuned model section, the user will see an improvements in the ranking metrics of the embeddings model. And in the retrieve similar items using the tuned embedding model you can explore how better are the results looking at the associated dataframe.

File: embeddings/get_started_with_embedding_tuning.ipynb [hunk 5]
```diff
@@ -1,1438 +1,1438 @@
...
+        "job.run()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "UJCufZmS7SVM"
+      },
+      "source": [
+        "### Evaluate the tuned model\n",
```
[the remark this answers is not in the document]
Comment (inardini) [reply embeddings/get_started_with_embedding_tuning.ipynb:1161]: add explanation from the coming blog post

**Review by polong-lin — APPROVED** [review]

**Review by polong-lin — APPROVED** [review]

**Review by holtskinner — APPROVED** [review]

**Pull request merged**